barretenberg_wasm/src/lib.rs

```diff
@@ -15,7 +15,6 @@ pub mod scalar_mul;
 pub mod schnorr;
 
 pub use common::crs;
-use std::cell::Cell;
```

Comment:
We make use of this on L80. Shouldn't we make that a qualified usage of `Cell` if we're removing this import?